lib/puppet/provider/foreman_resource/rest_v3.rb

```diff
@@ -95,6 +98,20 @@ def success?(response)
   end
 
   def error_message(response)
-    JSON.parse(response.body)['error']['full_messages'].join(' ') rescue "unknown error (response #{response.code})"
+    messages = {
+      '400' => '400 Bad Request: Something is wrong with the data we sent to Foreman server',
```

Comment:
'we' can be ambiguous when used in these kinds of contexts, could the message say "Something is wrong with the data sent to the Foreman server"

Reply:
The next version I push will remove the 'we'